cmd/buildah/commit.go

```diff
@@ -23,6 +23,7 @@ import (
 
 type commitInputOptions struct {
 	authfile           string
+	addHistory         bool
```

Comment:
Can you make this disableHistory for consistency or stick to add-history throughout.

Reply:
+1

Reply:
Made this `--add-history` throughout, since we discussed this before here: https://github.com/containers/buildah/issues/3513#issuecomment-975899755